glyphs-reader/src/font.rs

```diff
@@ -333,24 +335,84 @@ impl FromPlist for LayerAttributes {
             let key: String = tokenizer.parse()?;
             tokenizer.eat(b'=')?;
             match key.as_str() {
-                "coordinates" => {
-                    coordinates = tokenizer.parse()?;
-                }
+                "coordinates" => coordinates = tokenizer.parse()?,
+                "color" => color = tokenizer.parse::<i64>()? == 1,
                 // skip unsupported attributes for now
                 // TODO: match the others
                 _ => tokenizer.skip_rec()?,
             }
             tokenizer.eat(b';')?;
         }
 
-        Ok(LayerAttributes { coordinates })
+        Ok(LayerAttributes { coordinates, color })
+    }
+}
+
+#[derive(Clone, Default, Debug, PartialEq, Eq, Hash, FromPlist)]
+pub struct ShapeAttributes {
+    pub gradient: Gradient,
+}
+
+#[derive(Clone, Default, Debug, PartialEq, Eq, Hash, FromPlist)]
+pub struct Gradient {
+    start: Vec<OrderedFloat<f64>>,
+    end: Vec<OrderedFloat<f64>>,
+    colors: Vec<Color>,
+    #[fromplist(key = "type")]
+    style: String,
+}
+
+#[derive(Clone, Default, Debug, PartialEq, Eq, Hash)]
+pub struct Color {
+    pub r: i64,
+    pub g: i64,
+    pub b: i64,
+    pub a: i64,
+    pub n: i64, // 0/1 for start/end
```

Comment:
I think this would be better named `stop_offset`?

Reply:
And are the only valid values `0` and `1`? Would it make more sense as an enum?

Reply:
no no, it's not an enum, and neither it is an integer, it's a float that ranges from 0.0 (start) to 1.0 (end) and corresponds to COLRv1 StopOffset (along the 'color line'); if Rod only encountered whole 0 and 1 it's because he did not add intermediate stops

Reply:
That's right, this PR was only really meant to parse the one example I have so far and then incrementally grow more capable over time. That said, I have updated to stop_offset: f64.